packages/lib/src/toolbar/controls/DomainSlider/Thumb.tsx

```diff
@@ -10,18 +10,21 @@ interface Props extends HTMLProps<HTMLDivElement> {
   bound: Bound;
   isAuto: boolean;
   hasError: boolean;
+  disabled: boolean;
   AutoIcon: IconType;
 }
 
 const Thumb = forwardRef<HTMLDivElement, Props>((props, ref) => {
-  const { bound, isAuto, hasError, AutoIcon, ...thumbProps } = props;
+  const { bound, isAuto, hasError, disabled, AutoIcon, ...thumbProps } = props;
 
   return (
     <div
       ref={ref}
       {...thumbProps}
       className={styles.thumb}
       aria-label={`Change ${bound} limit`}
+      aria-disabled={disabled || undefined}
+      tabIndex={disabled ? -1 : thumbProps.tabIndex} // prevent tabbing when disabled
```

Comment:
Doing it myself because it's not done currently 😢 https://github.com/zillow/react-slider/pull/268